developments_build/sql/yearly.sql

```diff
@@ -110,10 +109,9 @@ SELECT a.job_number,
         CASE WHEN a.complete_year = '2019' AND a.job_inactive IS NULL
             THEN a.classa_net
             ELSE NULL END AS comp2019,
-        CASE WHEN a.complete_year = '2020' AND a.date_complete < '2020-07-01'::date
-                AND a.job_inactive IS NULL
+        CASE WHEN a.complete_year = '2020' AND a.job_inactive IS NULL
             THEN classa_net
-            ELSE NULL END AS comp2020q2,
+            ELSE NULL END AS comp2020,
         CASE WHEN a.date_complete > '2010-03-31'::date AND a.date_complete < '2020-07-01'::date
```

Comment:
should we change the `a.date_complete < '2020-07-01'::date` filter here too?

Reply:
maybe we can use the environmental variable to setup the filter?
https://github.com/NYCPlanning/db-developments/blob/64d9ecda789993e25fef4c797f8680a10172252d/developments_build/version.env#L1

Reply:
What would be the advantage of that if it is an entire year?

Reply:
See #321 and #323, if you are referring to since_cen10.